public/js2/kaiadmin.js

```
@@ -0,0 +1,406 @@
+				nav_open = 0;
+			}  else {
+				$('html').addClass('nav_open');
+				toggle.addClass('toggled');
+				nav_open = 1;
+			}
+		});
+		toggle_sidebar = true;
+	}
+
+	if(!quick_sidebar_open) {
+		var toggle = $('.quick-sidebar-toggler');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'toggle'. Consider to delete it or rename it.

'toggle' is defined here:


(lint/suspicious/noRedeclare)